### DISCUSSION TITLE
Can we add a wildcard domain while using plug and play in web3Auth?

### DISCUSSION THREAD

**Discussion by shahbaz17** [Web3Auth/Web3Auth#393] [Help]
Does web3auth supports wildcard domains?

**Comment by rinzler99** [comment]
Web3auth doesn't allow wildcards in redirect urls for a security reason.
Here's why in the OAuth2 spec: https://datatracker.ietf.org/doc/html/rfc6749#section-3.1.2

The redirection endpoint URI MUST be an absolute URI as defined by
   [RFC3986] Section 4.3.  The endpoint URI MAY include an
   "application/x-www-form-urlencoded" formatted (per Appendix B) query
   component ([RFC3986] Section 3.4), which MUST be retained when adding
   additional query parameters.  The endpoint URI MUST NOT include a
   fragment component.

**Comment by shahbaz17** [comment]
Also, answered https://github.com/orgs/Web3Auth/discussions/306.

**Comment by chaitanyapotti** [comment]
You can use the wildcard urls w/o using the dashboard.
https://github.com/orgs/Web3Auth/discussions/351
